### PR TITLE
py-htseq: update to 0.10.0; add Python 3.6

### PR DESCRIPTION
**Pull request by dliessi**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010
Xcode 8.2.1 8C1002

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-htseq/Portfile [hunk 1]
```diff
@@ -7,29 +7,31 @@ set _name           HTSeq
...
 distname            ${_name}-${version}
 
-checksums           rmd160  034ef4917faabb8659adc89ceb1ab0d94d7703bd \
-                    sha256  9bff6779595ea13c10137dad3bc4ecfd046b0bc620990eaa08920a29dcbd05c5
+checksums           rmd160  1a08aa9cd722f200e8ce2760eaaf879781ca68ff \
+                    sha256  e119af03cf068e981d7e08c4bd6393fd635007b595541943615b72abd676e547 \
+                    size    609412
 
-python.versions     27
+python.versions     27 36
```
Comment (pmetzger) [inline python/py-htseq/Portfile]: Shouldn't 3.7 also be added?

**Comment by pmetzger** [comment]
@reneeotten Could you have a look at the whole Portfile?

File: python/py-htseq/Portfile (hunk 1 above)
Comment (yan12125) [reply python/py-htseq/Portfile]: The dependency pysam is not yet 3.7-compatible in the latest release - https://github.com/pysam-developers/pysam/issues/697

Comment (pmetzger) [reply python/py-htseq/Portfile]: Then I think a comment should be added saying that when pysam is available 3.7 should be added.

Comment (dliessi) [reply python/py-htseq/Portfile]: Ok, I'll do it later.

**Comment by dliessi** [comment]
I updated the pull request, adding a comment on py-pysam and Python 3.7, as requested.

**Comment by pmetzger** [comment]
Merged. Thank you, @dliessi!

**Pull request merged**